src/visualization/types/SimpleTable/PagedTable.tsx

```diff
@@ -269,7 +271,7 @@ const PagedTable: FC<Props> = ({result, properties}) => {
         clearTimeout(timeout)
       }
     }
-  }, [ref?.current])
+  }, []) // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
`ref.current` should not be a dependency. See https://stackoverflow.com/questions/60476155/is-it-safe-to-use-ref-current-as-useeffects-dependency-when-ref-points-to-a-dom

Removing it leaves an empty dependency array, which means this hook runs only once. But he hook also has a cleanup function, so rest assured that the hook is cleaning up after itself when the SimpleTable is unmounted.